docs/DeveloperGuide.md

```diff
@@ -115,9 +115,9 @@ Here's a (partial) class diagram of the `Logic` component:
 
 <img src="images/LogicClassDiagram.png" width="550"/>
```

Comment:
The logicClassDiagram AddCommnad for the examples XYZCommand no longer exists may be better to change